lib/awful/titlebar.lua

```diff
@@ -513,6 +513,8 @@ local function new(c, args)
     -- Handle declarative/recursive widget container
     ret.setup = base.widget.setup
 
+    c.titlebars = bars
```

Comment:
Can this commit be squashed with whatever makes use of this property?
Put differently: I don't see any use of `c.titlebars` in the following commits.